security.tf

```diff
@@ -22,6 +22,17 @@ resource "aws_security_group" "sg" {
   tags = merge(local.tags, { "Name" : "${var.prefix}-sg" })
 }
 
+resource "aws_security_group_rule" "private_vpc" {
+  count             = var.vpc_private_endpoints_enabled ? length(local.cluster_endpoint_private_access_cidrs) > 0 ? 1 : 0 : 0
+  type              = "ingress"
+  description       = "Allow tcp port 443 ingress from all VPC endpoints"
```

Comment:
From Phil Hopkins:
the description of the new rule is a bit confusing, because the SG gets associated to VPC Endpoint ENIs which facilitate access TO the AWS Service which is the target of the VPC Endpoint, I think it should read:

description = "Allow tcp port 443 ingress to all AWS Services targeted by the VPC endpoints"

Reply:
applied